static/js/components/LearnerSearch.js

```diff
@@ -285,6 +287,7 @@ export default class LearnerSearch extends SearchkitComponent {
             field="profile.birth_country"
             operator="OR"
             itemComponent={CountryRefinementOption}
+            include='.+'
```

Comment:
I _think_ that makes it so the filter properly hides when there are no options to select, however, since implementation dragged out so long I'm not certain. I'll remove it for now.